src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -0,0 +1,110 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.Button;
+
+/**
+ * A wrapper class for our custom-made operator board. Does not have a
+ * constructor. Access statically. Access Buttons via public static state.
+ * Access axis values via methods. Use methods to make buttons light up.
+ */
+public class OperatorBoard {
+
+    private static Joystick HARDWARE;
+    public OperatorBoardButton BUTTON_ONE;
+    public OperatorBoardButton BUTTON_TWO;
+    public OperatorBoardButton BUTTON_THREE;
+    public OperatorBoardButton BUTTON_SIX;
+    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton BUTTON_TWELVE;
+    public OperatorBoardButton BUTTON_THIRTEEN;
+    public OperatorBoardButton BUTTON_FOURTEEN;
+    // TODO rename these, and use lowercase camel case names
```

Comment:
why not do now?

Reply:
i think she means when we implement it, so the buttons actually do things